src/Umbraco.Cms.Integrations.Crm.Hubspot/Umbraco.Cms.Integrations.Crm.Hubspot.csproj

```diff
@@ -10,21 +10,33 @@
 		<PackageIconUrl></PackageIconUrl>
 		<PackageProjectUrl>https://github.com/umbraco/Umbraco.Cms.Integrations/blob/main/src/Umbraco.Cms.Integrations.Crm.Hubspot</PackageProjectUrl>
 		<RepositoryUrl>https://github.com/umbraco/Umbraco.Cms.Integrations</RepositoryUrl>
-		<Version>1.1.5</Version>
+		<Version>1.1.6</Version>
 		<Authors>Umbraco HQ</Authors>
 		<Company>Umbraco</Company>
 		<PackageTags>Umbraco;Umbraco-Marketplace</PackageTags>
 		<PackageIcon>hubspot.png</PackageIcon>
 	</PropertyGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)' == 'net472'">
-		<PackageReference Include="UmbracoCms.Web" version="8.4.0" />
+		<PackageReference Include="UmbracoCms.Web" version="[8.4.0, 9)" />
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)' == 'net50'">
-		<PackageReference Include="Umbraco.Cms.Web.Website" version="9.0.1" />
-		<PackageReference Include="Umbraco.Cms.Web.BackOffice" version="9.0.1" />
-		<PackageReference Include="Umbraco.Cms.Web.Common" version="9.0.1" />
+		<PackageReference Include="Umbraco.Cms.Web.Website" version="[9.0.1,10)" />
+		<PackageReference Include="Umbraco.Cms.Web.BackOffice" version="[9.0.1,10)" />
+		<PackageReference Include="Umbraco.Cms.Web.Common" version="[9.0.1,10)" />
+	</ItemGroup>
+	
+	<ItemGroup Condition="'$(TargetFramework)' == 'net60'">
+		<PackageReference Include="Umbraco.Cms.Web.Website" version="[10.0.0,12)" />
```

Comment:
I think this can actually be `[10.0.0,11)`.  As Umbraco 11 only runs on .NET 7.  Check against the package I shared with you though to verify how it's specified there.